test/forwarder.test.js

```diff
@@ -49,27 +49,16 @@ class Provider {
     }
 }
 
-describe('::json-rpc-forwarder', function () {
+describe('::forwarder', function () {
     /** @type {Provider} */
     let provider;
 
     before(async function () {
-        const scriptPath = resolve(__dirname, '..', '..', 'src', 'plugin', 'json-rpc-forwarder');
-        const port = await new Promise(resolve =>
-            new Worker(scriptPath, {
-                workerData: {
-                    mirrorNodeUrl: '',
-                    localAccounts: ['0x70997970c51812dc3a010c7d01b50e0d17dc79c8'],
-                },
-            })
-                .on('message', message => {
-                    if (message.listening) {
-                        resolve(message.port);
-                    }
-                })
-                .unref()
-        );
-        provider = new Provider(port);
+        const forkUrl = /**@type{string}*/ (/**@type{unknown}*/ (undefined));
+        const { host } = await jsonRPCForwarder(forkUrl, '', undefined, [
+            '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
```

Comment:
Just out of curiosity, what's this address? A random non-existing account or a local node account?

Reply:
These are local Hardhat accounts. It's used as an optimization. The forwarder won't forward any rpc call related to these accounts, because it knows they are local hence there should be no remote state to fetch.